cli/version.go

```diff
@@ -27,7 +27,7 @@ var versionCmd = &cobra.Command{
 	Short: "Print sedge version",
 	Long:  ``,
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Fprintln(cmd.OutOrStdout(), "sedge 0.1")
+		fmt.Fprintln(cmd.OutOrStdout(), "sedge v0.2.0")
```

Comment:
Any way to store sedge version somewhere and bump up after code changes, patches, etc? Might be worth looking into as future versions should be released, with the changelog available it should be good.